packages/ifpack2/src/Ifpack2_Relaxation_def.hpp

```diff
@@ -143,10 +143,22 @@ namespace {
       return Teuchos::ScalarTraits<Scalar>::eps ();
     }
   };
+
+
+
 } // namespace (anonymous)
 
 namespace Ifpack2 {
 
+template<class MatrixType>
+void Relaxation<MatrixType>::updateCachedMultiVector(const Teuchos::RCP<const Tpetra::Map<local_ordinal_type,global_ordinal_type,node_type> > & map, size_t numVecs) const{
```

Comment:
Should this be a reference to an RCP?

Reply:
Nevermind.  Didn't scroll over far enough, ugh.

Reply:
Yes.  Crud.